src/components/Title.tsx

```diff
@@ -0,0 +1,16 @@
+type stringTitle = {
+  Titlename: string;
+};
+
+const Heading = ({ Titlename }: stringTitle) => {
```

Comment:
```suggestion
const Title = ({ Titlename }: stringTitle) => {
```
the component you name here should match the file name